packages/flutter/lib/src/widgets/autofill.dart

```diff
@@ -225,3 +228,142 @@ class _AutofillScope extends InheritedWidget {
   @override
   bool updateShouldNotify(_AutofillScope old) => _scope != old._scope;
 }
+
+/// An [InheritedWidget] that configures its descendant [Navigator]s and [Form]s,
+/// as to whether the current autofill context should be saved or discarded, or
+/// no action should be taken at all, when any of the [Navigator]s' topmost
+/// opaque [ModalRoute]s changes, or when [FormState.save] is called on any of
+/// the [Form]s.
+///
+/// {@macro flutter.services.autofill.autofillContext}
+///
+/// Set [AutofillContextLifecycleAction.globalLifecycleDelegate] if you wish to
+/// change the default [AutofillContextLifecycleDelegate] for the entire app.
+///
+/// The default [globalLifecycleDelegate] saves the current autofill context
```

Comment:
Having this global is odd. Why do we need it? If you want different settings, you'd just wrap your subtree in a different `AutofillContextLifecycleAction` widget.